jck/devtools.java2schema/playlist.xml

```diff
@@ -17,14 +17,14 @@
 	<test>
 		<testCaseName>jck-devtools-java2schema-CustomizedMapping</testCaseName>
 		<disabled>
-			<comment>Disabled on x64 Linux + aarch64 + ppc64le + win64 on jdk8 due to backlog/issues/302. Awaiting potential test / setup issue resolution</comment>
-			<platform>x86-64_linux(_mixed)?|aarch64_linux(_mixed)?|ppc64le_linux(_mixed)?|x86-64_windows(_mixed)?</platform>
+			<comment>Disabled on x64 Linux + aarch64 + ppc64le on jdk8 due to backlog/issues/302. Awaiting potential test / setup issue resolution</comment>
```

Comment:
Has `backlog/issues/302` been resolved along with `backlog/issues/507`?
If so, other platforms can be un-excluded as well.

Reply:
> Has backlog/issues/302 been resolved along with backlog/issues/507? 

No,  backlog/issues/507 only fixes the issue with the batch scripts, the *.sh script issue (backlog/issues/302) still remains. We'd investigate / resolve / un-exclude tests for that issue separately.

Reply:
Had an offline chat with Mesbah, the initial windows failure was due to `backlog/issues/507` though they were excluded under a broader `backlog/issues/302`.